libbeat/docs/http-endpoint.asciidoc

```diff
@@ -56,7 +56,7 @@ curl -XGET 'localhost:5066/?pretty'
 ["source","js",subs="attributes"]
 ----
 {
-  "beat": "metricbeat",
+  "beat": "heartbeat",
```

Comment:
Umm, I think this should be `{beatname_lc}`, so this is replaced by the lowercased name of the actual beat on each guide.
```suggestion
  "beat": "{beatname_lc}",
```